org.adempiere.base/src/org/compiere/print/layout/LayoutEngine.java

```diff
@@ -112,21 +112,26 @@ public class LayoutEngine implements Pageable, Printable, Doc
 	 *  @param format Print Format
 	 *  @param data Print Data
 	 *  @param query query for parameter info
+	 *  @param info
+	 *  @param windowNo
 	 */
-	public LayoutEngine (MPrintFormat format, PrintData data, MQuery query, PrintInfo info )
+	public LayoutEngine (MPrintFormat format, PrintData data, MQuery query, PrintInfo info, int windowNo )
 	{
-		this(format, data, query, info , null);
+		this(format, data, query, info , null, windowNo);
 	}	//	LayoutEngine
 	
 	/**
 	 *	Detail Constructor
 	 *  @param format Print Format
 	 *  @param data Print Data
 	 *  @param query query for parameter info
+	 *  @param info
 	 *  @param trxName
+	 *  @param windowNo
 	 */
-	public LayoutEngine (MPrintFormat format, PrintData data, MQuery query, PrintInfo info ,  String trxName)
+	public LayoutEngine (MPrintFormat format, PrintData data, MQuery query, PrintInfo info ,  String trxName, int windowNo)

```

Comment:
for backward compatibility, please keep the 2 existing LayoutEngine public constructor.

Reply:
This is resolved